### PR TITLE
Development

### PR DESCRIPTION
**Pull request by newcoder171**

### CODE REVIEW

**Comment by dipto0321** [comment]
Hi @newcoder171 and @mumani ,

You guys did a nice job 👍 . Just a couple of things you need to fix which I write down 👇 below:

1. Please try to use markdown to give a better look to your readme. You already give better info into readme. Just add a demo link of your site. 

2. [It does not look like the original site](https://prnt.sc/m9juqq). So try to fix it. It will be a little bit tricky. But trust me, it will be fun. I know that you can do it. You have that ability 😃 🍾 🎉 .

3. ` Bad value ./image/NYT Logo.png for attribute src on element img: Illegal character in path segment: space is not allowed.` You can find a lot of error's like this when you will check your site with w3validator. So check it and fix all issues.

4. Put your `main.css` file inside `css` folder and link it to your `.html` file.

5. There is some merge conflict issue with your master branch. So after fixing this or before fixing try to resolve it.

I think except those ☝️ all are good. The code is clean with indentation. Try to build a habit of comment ( should be meaningful ) in your code from your future project.

Please use [w3validator](https://validator.w3.org) and [CSS lint](http://csslint.net/) and try to fix all issues.

**Note:** After fixing, I think you can merge it into the master or if you want to review it again then make e review request again. Best of luck 😄

File: index.html [hunk 1]
```diff
@@ -1,10 +1,521 @@
...
+        <p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of
+            Cosmic
+            Extragalactic Polarization, reported their results in a scientific briefing at the Center for
+            Astrophysics
+            here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+        <!-- Image excerpt start-->
+        <figure>
+            <img src="https://static01.nyt.com/newsgraphics/2014/03/18/theory-of-inflation/1ebaa1b972edda0f407fbaa0e010f6b50fb6792e/inflation-720.png">
+            <div class="inflation-fig1">
+                <figcaption><strong>THE UNIVERSE</strong> is just under 14 billion years old. From our position in the
```
Comment (heyapricot) [inline index.html:96]: Element `figcaption` not allowed as child of element div in this context. Fix all the occurrences

File: index.html [hunk 2]
```diff
@@ -1,10 +1,521 @@
...
+        <p>By LARRY BUCHANAN and JONATHAN CORUM</p>
+        <p>Dr. Kovac said the chance that the results were a fluke was only one in 10 million.</p>
+        <p>Dr. Guth, now 67, pronounced himself “bowled over,” saying he had not expected such a definite
+            confirmation
+            in his lifetime.</p>
+
+        <!-- Side Items -->
+        <aside id="editor-pick">
+            <ul>
+                <span>Editors' Pick</span>
```
Comment (heyapricot) [inline index.html]: Element `span` not allowed as child of element `ul` in this context

File: index.html [hunk 3]
```diff
@@ -1,10 +1,521 @@
...
+                <hr>
```
Comment (heyapricot) [inline index.html]: Element `hr` not allowed as child of element `ul` in this context.

File: index.html [hunk 4]
```diff
@@ -1,10 +1,521 @@
...
+            which has operated a series of increasingly sensitive radio telescopes at the South Pole, where the
+            thin, dry
+            air creates ideal observing conditions. The others are Clement Pryke of the University of Minnesota,
+            Jamie Bock
+            of the California Institute of Technology and Dr. Kuo of Stanford.</p>
+        <p>“The South Pole is the closest you can get to space and still be on the ground,” Dr. Kovac said. He has
+            been
+            there 23 times, he said, wintering over in 1994. “I’ve been hooked ever since,” he said.</p>
+        <!-- Illustration Image start -->
+        <img alt="" class="img5" src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos-2/18cosmos-2-articleLarge.jpg?"
```
Comment (heyapricot) [inline index.html:220]: Here, for the `width` property you don't have to add `px`. Remove `px`

File: index.html [hunk 5]
```diff
@@ -1,10 +1,521 @@
...
+
+        </div>
+        <!-- Trending news section ends   -->
+
+    </div>
+    <div class="clear hidden">A</div>
+
+    <footer>
+        <hr style="margin-bottom: 2px">
+        <img src="./image/NYT Logo.png" alt="logo" height="25">
```
Comment (heyapricot) [inline index.html]: `space` is not allowed in paths. Check the `./image/NYT Logo.png` path

File: index.html [hunk 6]
```diff
@@ -1,10 +1,521 @@
...
+            <ul>
+                <li>SUBSCRIBE</li>
+                <li class="bold-text"><i class="material-icons">local_dining</i>Home Delivery</li>
+                <li class="bold-text"><i class="material-icons">local_dining</i>Digital Subscriptions</li>
+                <li class="bold-text"><i class="material-icons">local_dining</i>Crossword</li>
+                <li class="bold-text"><i class="material-icons">local_dining</i> Cooking</li>
+                <li class="small-text">Email Newsletters</li>
+                <li class="small-text">Corporate Subscriptions</li>
+                <li class="small-text">Education Rate</li>
+                <hr style="margin-bottom: 8px">
```
Comment (heyapricot) [inline index.html:491]: Element `hr` not allowed as child of element `ul` in this context.

File: css/main.css [hunk 7]
```diff
@@ -0,0 +1,341 @@
...
+.small-text {
+    font-weight: normal;
+    font-size: 11px;
+}
+
+li>i {
+    margin: 5px;
+}
+
+hr {
```
Comment (heyapricot) [inline css/main.css:293]: Don't duplicate selectors. You used this one first on line 82

File: index.html [hunk 8]
```diff
@@ -1,10 +1,521 @@
...
+            <i class="material-icons">menu</i>
+            <i class="material-icons">search</i>
+            <span>SPACE & COSMOS</span>
+            <img src="./image/NYT-Logo.png" alt="logo" height="25">
+            <button>SUBSCRIBE NOW</button>
+            <button>LOG IN</button>
+        </nav>
+    </header>
+
+    <hr>
```
Comment (heyapricot) [inline index.html:26]: Is this `hr` element doing anything?

File: index.html [hunk 9]
```diff
@@ -1,10 +1,521 @@
...
+    <section>
+        <!-- section div start-->
+        <span id="headline">Space Ripples Reveal <br> Big Bang’s Smoking Gun</span>
+        <div>
```
Comment (heyapricot) [inline index.html:30]: The width of this element is different from the original

File: index.html [hunk 10]
```diff
@@ -1,10 +1,521 @@
...
+                alt="" width="900">
+            <p class="img1">
+                Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains
+                how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8
+                billion
+                years ago.CreditCreditRick Friedman for The New York Times</p>
+        </div>
+        <span style="font-weight: 700">By Dennis Overbye</span>
+        <div>
+            <p style="text-align: center">March 17, 2014</p>
```
Comment (heyapricot) [inline index.html:41]: The author and the date are not aligned to the center in the original article. Also, watch the spaces (And don't use `br` for spacing. Use CSS)

File: index.html [hunk 11]
```diff
@@ -1,10 +1,521 @@
...
+            whether it could ever be proved.</p>
+        <p>If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery
+            of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time
+            and space and energy to science and speculation.</p>
+        <p>Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with
+            its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent,
+            architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless
+            number
+            of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
+        <h2><strong>‘As Big as It Gets’</strong></h2>
```
Comment (heyapricot) [inline index.html:75]: The original has space in the top and the bottom

File: index.html [hunk 12]
```diff
@@ -1,10 +1,521 @@
...
+            expressed reservations about inflation.</p>
+        <p>Lawrence M. Krauss of Arizona State and others also emphasized the need for confirmation, noting that
+            the
+            new results exceeded earlier estimates based on temperature maps of the cosmic background by the
+            European
+            Space Agency’s Planck satellite and other assumptions about the universe.</p>
+        <p>“So we will need to wait and see before we jump up and down,” Dr. Krauss said.</p>
+        <p>Corroboration might not be long in coming. The Planck spacecraft will report its own findings this year.
+            At least a dozen other teams are trying similar measurements from balloons, mountaintops and space.</p>
+        <p>Gravity waves are the latest and deepest secret yet pried out of the cosmic microwaves, which were
```
Comment (heyapricot) [inline index.html:206]: You missed a header before this

File: index.html (hunk 4 above)
Comment (heyapricot) [reply index.html:220]: Also, make the width match the origina

File: index.html [hunk 13]
```diff
@@ -1,10 +1,521 @@
...
+            have
+            little arrows going in spirals.</p>
+        <p>Detecting those spirals required measuring infinitesimally small differences in the temperature of the
+            microwaves.
+            The group’s telescope, Bicep2, is basically a giant superconducting thermometer.</p>
+        <p>“We had no expectations what we would see,” Dr. Kovac said.</p>
+        <p>The strength of the signal surprised the researchers, and they spent a year burning up time on a Harvard
+            supercomputer,
+            making sure they had things right and worrying that competitors might beat them to the breakthrough.</p>
+        <p>A Special Time</p>
```
Comment (heyapricot) [inline index.html:240]: This should be a header

File: index.html [hunk 14]
```diff
@@ -1,10 +1,521 @@
...
+            of the team,
+            “you wind up at inflation.”</p>
+        <p>A version of this article appears in print on March 18, 2014, on Page A1 of the New York edition with
+            the headline:
+            Space Ripples Reveal Big Bang’s Smoking Gun.<a href=""> Order Reprints </a>|<a href=""> Today’s Paper
+            </a>|<a href="">
+                Subscribe</a></p>
+
+        <div class="btm-btns">
+            <button class="s-icons"><i class="material-icons">reply</i></button>
```
Comment (heyapricot) [inline index.html:302]: Icons are not centered

File: index.html [hunk 15]
```diff
@@ -1,10 +1,521 @@
...
+            <button class="s-icons"><i class="material-icons">mail</i></button>
+            <button class="s-icons" style="font-size: 23px">t</button>
+            <button class="s-icons" style="font-size: 23px">f</button>
+            <button>READ 615 COMMENTS</button>
+            <br /><br /><br>
```
Comment (heyapricot) [inline index.html]: Don't add multiple `br` elements for spacing. If you want to add spacing do it through CSS

File: index.html [hunk 16]
```diff
@@ -1,10 +1,521 @@
...
+                <li>Photography</li>
+                <li>Video</li>
+                <li>NYT Store</li>
+                <li>Times Journeys</li>
+                <li>Manage My Account</li>
+            </ul>
+        </div>
+
+
+        <div class="footer">
```
Comment (heyapricot) [inline index.html:481]: You are missing the NY Times image, the font should be sans-serif and the icons are repeated

**Review by heyapricot — CHANGES_REQUESTED** [review]
# You are almost there
So close! There is a good amount of details that need to be fixed so I have to ask you again to work on those and submit a new code review request when you are finished. Thank you for your perseverance! Please address the comments, and after that submit a new code review request.

Additional comments:
Check the alignment of the navbar's content
screen shot 2019-01-29 at 18 15 29

Check the alignment of this section
screen shot 2019-01-29 at 18 24 02



Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

**Comment by dipto0321** [comment]
Hi @newcoder171  and @mumani ,
1. Please check these [errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fmumani%2FNY-Times%2Fdevelopment%2Findex.html). and fix it. Again check with [w3Validator](https://validator.w3.org/).
2. I found some resoulation based problem. especially on the alignment with the nav bar logo as mentioned before. (Use the zoom in and out in your browser to understand the problem). 
nav_portion
big_portion

After those changes, you can merge it into master. Don't need to request for any code review. But if you want to you can.

Let me know if you have any question.

Thanks

**Pull request merged**